modules/gdscript/gdscript_codegen.h

```diff
@@ -110,6 +109,8 @@ class GDScriptCodeGenerator {
 	virtual void write_get_named(const Address &p_target, const StringName &p_name, const Address &p_source) = 0;
 	virtual void write_set_member(const Address &p_value, const StringName &p_name) = 0;
 	virtual void write_get_member(const Address &p_target, const StringName &p_name) = 0;
+	virtual void write_set_static_variable(const Address &p_class, const Address &p_index, const Address &p_value) = 0;
+	virtual void write_get_static_variable(const Address &p_class, const Address &p_index, const Address &p_dst) = 0;
```

Comment:
The index can be simply `int`, it does not need to be a full `Address` because it will be always be known at compile-time.